src/sage/combinat/combinat.py

```diff
@@ -2963,6 +2965,12 @@ def bell_polynomial(n: Integer, k: Integer):
         [[], [1], [3], [1]]
         [[], [1], [3, 4], [6], [1]]
 
+   further checks for :issue:`37727`::
+        sage: bell_polynomial(0,0)                                                      # needs sage.combinat
+        1
+        sage: bell_polynomial(1,1)                                                      # needs sage.combinat
+        x0
+
```

Comment:
```suggestion
        x0
```
One too many blanklines.